pom.xml

```diff
@@ -23,7 +23,7 @@
   </licenses>
 
   <properties>
-    <bioformats.version>5.9.0</bioformats.version>
+    <bioformats.version>5.9.1-SNAPSHOT</bioformats.version>
```

Comment:
This version increase failed since there is no 5.9.1-SNAPSHOT artifact published to our Artifactory yet. Generally, following other components, I would vote  for keeping the dependency version set to non-SNAPSHOT versions e.g. stay at 5.9.0 especially in the `master` branch.
For coupling builds, bumping the versions to SNAPSHOT should simply become then the responsibility of https://github.com/ome/bio-formats-build/blob/master/scripts/update-versions.